manifests/nodejs.yml

```diff
@@ -575,7 +575,7 @@ tests/:
     test_stats.py:
       Test_Client_Stats: missing_feature
   test_config_consistency.py:
-    Test_Config_ClientIPHeader_Configured: *ref_4_0_0
+    Test_Config_ClientIPHeader_Configured: *ref_3_13_0
```

Comment:
well now you forgot to remove the ref_4.0.0 at the top of the file, that you added in the previous PR but that is now unused right ?

Reply:
Thanks for calling this out. I removed it!